resym/src/resym_app.rs

```diff
@@ -152,6 +152,28 @@ impl eframe::App for ResymApp {
 
             self.update_code_view(ui);
         });
+
+        ctx.input(|i| {
+            // Handle dropped files
+            if !i.raw.dropped_files.is_empty() {
+                // Allow dropping 1 file (to just view it), or 2 files to diff them
+                let slots = vec![PDB_MAIN_SLOT, PDB_DIFF_SLOT];
```

Comment:
I received some feedback on this that I'll implement, please hold off committing this!

Reply:
Implemented.